templates/taxi/index.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>Taxi Service</h1>
+    <p>Welcome to great Taxi Service!</p>
+    <h3>Taxi content:</h3>
+    <ul>
+        <li>Number of drivers: {{ num_drivers }}</li>
+        <li>Number of manufacturers: {{ num_manufacturers }}</li>
+        <li>Number of cars: {{ num_cars }}</li>
+    </ul>
+{% endblock %}
```

Comment:
Please, add a blank line at the end of the file

Reply:
Fixed. I thought it was not necessary because it is not a python file